examples/kotlin-kts/build.gradle.kts

```diff
@@ -8,7 +8,7 @@ import org.jetbrains.kotlin.gradle.tasks.*
 plugins {
     java
     kotlin("jvm")
```

Comment:
Applying both KGP and a Java plugin causes issues and is deprecated, so remove Java and the `tasks.withType<JavaCompile> {` config below.

https://kotlinlang.org/docs/whatsnew2020.html#deprecated-compatibility-with-kotlin-multiplatform-gradle-plugin-and-gradle-java-plugins

Reply:
IIUC, simultaneous usage of KMP and Java plugin is deprecated. Whereas, here kotlin-jvm plugin is used. Please correct me if I got it wrong.

Anyways, the `java` plugin is redundant here and is removed in a followup PR: https://github.com/Kotlin/kotlinx-benchmark/pull/257

Reply:
You're right! My mistake.